src/loaders/FileLoader.js

```diff
@@ -196,6 +196,14 @@ class FileLoader extends Loader {
 				// Abort errors and other errors are handled the same
 
 				const callbacks = loading[ url ];
+
+				if ( ! callbacks ) {
+
+					// When onLoad was called and url was deleted in `loading`
+					throw err;
```

Comment:
I guess it would make sense to call `this.manager.itemError( url );` before the `throw` statement. `itemEnd()` was already called in the previous `then()` implementation.

BTW: Please write the if clause like so:
```js
if ( callbacks === undefined )
```

Reply:
Done! So excited to finish my first pull request! :)